src/types/student/student.enum.ts

```diff
@@ -0,0 +1,16 @@
+export enum TypeWork {
+  'Praca w biurze' = 1,
```

Comment:
Ale... po co tak? Po co `= 1`, `= 2`, a nawet jak potrzebny, to wystarczy `= 1`, reszta sama się ustawi.